MAUI/Maps/data-labels.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title: Data Labels in MAUI Maps widget | Syncfusion
+description: Learn here all about the Data Labels feature of Syncfusion MAUI Maps (SfMaps) widget to add labels to the shapes and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Data Labels in MAUI Maps (SfMaps)
+
+Data labels provides identification for the shapes by displaying their names. You can trim or hide the labels if they exceed the shape bounds.
+
+## Show data labels
+
+You can show data labels on the map using the [`MapShapeLayer.ShowDataLabels`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeLayer/showDataLabels.html) property. By default, the data labels are rendered based on the value of [`ShapeDataField`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeSource/shapeDataField.html) property. The default value of the [`ShowDataLabels`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeLayer/showDataLabels.html) property is `false`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <map:SfMaps>
+        <map:SfMaps.Layer>
+            <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json"  DataSource="{Binding Data}" PrimaryValuePath="Continent" ShapeDataField="continent" ShowDataLabels = "True"  >
+
+                <map:MapShapeLayer.DataLabelSettings>
+                    <map:MapDataLabelSettings   DataLabelPath="Continent"/>
+                </map:MapShapeLayer.DataLabelSettings>
+
+            </map:MapShapeLayer>
+        </map:SfMaps.Layer>
+    </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+public MainPage()
+{
+    InitializeComponent();
+    ObservableCollection<Model> Data = new ObservableCollection<Model>();
+    Data.Add(new Model("Asia", "Asia"));
+    Data.Add(new Model("Europe", "EU"));
+    Data.Add(new Model("North America", "NA"));
+    Data.Add(new Model("South America", "SA"));
+    Data.Add(new Model("Australia", "Australia"));
+    Data.Add(new Model("Africa", "Africa"));
+    SfMaps maps = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromResource("world-map.json");
+    layer.DataSource = Data;
+    layer.PrimaryValuePath = "Continent";
+    layer.ShapeDataField = "continent";
+    layer.ShowDataLabels = true;
+
+    layer.DataLabelSettings = new MapDataLabelSettings()
+    {
+       DataLabelPath = "Continent",
+    };
+    maps.Layer = layer;
+    this.Content = maps;
+}
+public class Model
+{
+    public Model(string continent, string continentCode)
+    {
+        Continent = continent;
+        ContinentCode = continentCode;
+    }
+     public String Continent { get; set; }
+     public String ContinentCode { get; set; }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Data labels support](images/data-labels/default-data-labels.png)
```

Comment:
I have asked you to don't use a world map with continent data. The output will image is not fine. Please change it.

Reply:
Changed sir